client/src/components/navigations/MainStackNavigator/index.tsx

```diff
@@ -126,45 +108,10 @@ const onMessageSubscription = graphql`
 
 function MainStackNavigator(): ReactElement {
   const {theme} = useTheme();
-  // const currentAppState = useAppState();
   const navigation = useNavigation<MainStackNavigationProps<'MainTab'>>();
-  const environment = useRelayEnvironment();
-
-  const latestNavigation = useRef(navigation);
-  const latestEnvironment = useRef(environment);
 
   useEffect(() => {
-    requestPermissionsAsync();
-
-    const subscription = Notifications.addNotificationResponseReceivedListener(
-      async (response) => {
-        Notifications.setBadgeCountAsync(0);
-
-        const channelId = JSON.parse(
-          response.notification.request.content.data.data as string,
-        ).channelId;
-
-        fetchQuery<MainStackNavigatorChannelQuery>(
-          latestEnvironment.current,
-          channelQuery,
-          {
-            channelId,
-          },
-        ).subscribe({
-          next: (data) => {
-            if (data.channel)
-              latestNavigation.current.navigate('Message', {
-                channel: data.channel as Channel,
-                users: data.channel?.memberships?.map(
-                  (membership) => membership?.user as User,
-                ),
-              });
-          },
```

Comment:
@0916dhkim I think we need this part somewhere to direct to proper [Message] screen when notification is received then pressed.